jupyter_accessibility/dodo.py

```diff
@@ -37,7 +25,6 @@ def do(*args, cwd=A11Y, **kwargs):
     kwargs["env"] = {**environ, **kwargs["env"]}
     return CmdAction(list(map(str, args)), shell=False, cwd=str(Path(cwd).resolve()), **kwargs)
 
-
 def remove_directory(*dir):
     """remove a directory during the clean stage"""
     from shutil import rmtree
```

Comment:
Why is this import inside the function and not at the beginning of the file?

Reply:
Copy-paste. Will move to the top.